server/server.go

```diff
@@ -258,6 +260,7 @@ func newServer(ctx context.Context, c Config, rotationStrategy rotationStrategy)
 	handleFunc("/callback/{connector}", s.handleConnectorCallback)
 	handleFunc("/approval", s.handleApproval)
 	handleFunc("/healthz", s.handleHealth)
+	r.Handle("/metrics", promhttp.HandlerFor(prometheusRegistry, promhttp.HandlerOpts{}))
```

Comment:
needs to use `handleFunc`, which uses the correct relative paths.

Reply:
got it, there are actually some low hanging fruit here, I'll also instrument the http handlers